Makefile

```diff
@@ -331,7 +331,8 @@ changelog: ## Generate changelog
 	$(shell cat $(TMPFILE) >> changelog.txt)
 	$(shell rm $(TMPFILE))
 
-install: .gopathok install.bin install.remote install.man install.cni install.systemd  ## Install binaries to system locations
+install: podman podman-remote .gopathok install.bin install.remote install.man install.cni install.systemd  ## Install binaries to system locations
```

Comment:
Wouldn't it make more sense to make 
podman required for install.bin and podman-remite required for install.remote?

Reply:
i dont know, does it?

Reply:
I think it makes much more sense to move them there, yes.

Reply:
i guess my problem is I dont understand what you are asking for ... can you explain and i will make it so

Reply:
```
git diff 
diff --git a/Makefile b/Makefile
index 29eb9b3a..dbc10aa8 100644
--- a/Makefile
+++ b/Makefile
@@ -333,12 +333,12 @@ changelog: ## Generate changelog
 
 install: .gopathok install.bin install.remote install.man install.cni install.systemd  ## Install binaries to system locations
 
-install.remote:
+install.remote: podman-remote
        install ${SELINUXOPT} -d -m 755 $(DESTDIR)$(BINDIR)
        install ${SELINUXOPT} -m 755 bin/podman-remote $(DESTDIR)$(BINDIR)/podman-remote
        test -z "${SELINUXOPT}" || chcon --verbose --reference=$(DESTDIR)$(BINDIR)/podman bin/podman-remote
 
-install.bin:
+install.bin: podman
        install ${SELINUXOPT} -d -m 755 $(DESTDIR)$(BINDIR)
        install ${SELINUXOPT} -m 755 bin/podman $(DESTDIR)$(BINDIR)/podman
        test -z "${SELINUXOPT}" || chcon --verbose --reference=$(DESTDIR)$(BINDIR)/podman bin/podman
```

Reply:
ty! much much better